sdk/eventhub/Azure.Messaging.EventHubs/tests/Amqp/AmqpClientTests.cs

```diff
@@ -75,7 +76,9 @@ public void ConstructorRequiresTheCredential()
         [Test]
         public void ConstructorRequiresTheOptions()
         {
-            Assert.That(() => new AmqpClient("my.eventhub.com", "somePath", Mock.Of<TokenCredential>(), null), Throws.InstanceOf<ArgumentException>());
+            var credential = new Mock<EventHubTokenCredential>(Mock.Of<TokenCredential>(), "{namespace}.servicebus.windows.net");
+
```

Comment:
(very minor) nit:  I would close up the blank between the credential declaration and the client declaration, but I've no issue if you decide to leave it as-is.